fact_extractor/install/unpacker.py

```diff
@@ -154,44 +155,6 @@
             # 7z
             'yasm',
         ],
-        'pip3': [
-            'pluginbase',
-            'git+https://github.com/armbues/python-entropy',  # To be checked. Original dependency was deleted.
```

Comment:
?

Reply:
I could not find any usages (was probably in the entropy computation that is now done in common_helper_unpacking_classifier by https://github.com/fkie-cad/entropython)